vue.config.js

```diff
@@ -1,7 +1,7 @@
 const path = require('path');
 
 module.exports = {
-  publicPath: process.env.DEPLOY ? '././' : '/',
+  publicPath: process.env.PINATA ? './' : (process.env.DEPLOY ? '././' : '/'),
```

Comment:
`DEPLOY` env is not being used anymore, you can do this to simplify, i've tested it's working for all the cases.
```suggestion
  publicPath: './',
```

Reply:
Awesome, thanks :)